src/expander_pipeline.jl

```diff
@@ -524,8 +524,12 @@ function Selectors.runner(::Type{Expanders.AutoDocsBlocks}, node, page, doc)
         for mod in modules
             for (binding, multidoc) in Documenter.DocSystem.getmeta(mod)
                 # Which bindings should be included?
-                isexported = Base.isexported(mod, binding.var)
-                included = (isexported && public) || (!isexported && private)
+                ispublic = if isdefined(Base, :ispublic) # Julia v1.11 and later
```

Comment:
This could potentially be
```suggestion
                ispublic = @static if isdefined(Base, :ispublic) # Julia v1.11 and later